Source/Xamarin/Prism.Forms/Modularity/ModuleInfo.cs

```diff
@@ -0,0 +1,137 @@
+using System;
+using System.Collections.ObjectModel;
+using System.Reflection;
+
+namespace Prism.Modularity
+{
+    /// <summary>
+    /// Defines the metadata that describes a module.
+    /// </summary>
+    public partial class ModuleInfo : IModuleCatalogItem, IModuleInfo
```

Comment:
Doesn't IModuleInfo implement IModuleCatalogItem? So no need to declare them both?

Reply:
forgot to pull it off that one